visualizer/flame-graph.js

```diff
@@ -126,7 +126,7 @@ class FlameGraph extends HtmlContent {
   }
 
   initializeFromData () {
-    const { dataTree } = this.ui
+    const { dataTree, flameWrapper } = this.ui
```

Comment:
`this.ui.flameWrapper` is the instance of FlameGraph (i.e. `this` here), so we can skip this and use `this.d3Element` instead of `flameWrapper.d3Element`